ding/utils/deprecation.py

```diff
@@ -0,0 +1,57 @@
+import functools
+import textwrap
+import warnings
+
+
+def deprecated(since, removed_in, up_to=None):
+    """Decorate a function to signify its deprecation.
```

Comment:
modify the format of comments like our previous format ([example](https://github.com/opendilab/DI-engine/blob/main/ding/utils/default_helper.py#L11))